app/controllers/idv/unavailable_controller.rb

```diff
@@ -5,16 +5,7 @@ class UnavailableController < ApplicationController
     before_action :redirect_if_idv_available_and_from_create_account
 
     def show
-      analytics.vendor_outage(
-        vendor_status: {
-          acuant: IdentityConfig.store.vendor_status_acuant,
-          lexisnexis_instant_verify: IdentityConfig.store.vendor_status_lexisnexis_instant_verify,
-          lexisnexis_trueid: IdentityConfig.store.vendor_status_lexisnexis_trueid,
-          sms: IdentityConfig.store.vendor_status_sms,
-          voice: IdentityConfig.store.vendor_status_voice,
-        },
```

Comment:
I noticed a lot of this code was repeated from the code inside `outage_status.rb` so I consolidated